pootle/apps/pootle_app/management/commands/update_stores.py

```diff
@@ -56,8 +56,8 @@ def handle_store(self, store, **options):
         disk_mtime = store.get_file_mtime()
         if not options["force"] and disk_mtime == store.file_mtime:
             # The file on disk wasn't changed since the last sync
-            logging.debug(u"File didn't change since last sync, skipping "
-                          u"%s" % store.pootle_path)
+            logging.debug(u"File didn't change since last sync, "
+                          "skipping %s", store.pootle_path)
```

Comment:
Perhaps it is not a requirement (I thought it was but couldn't find anything supporting this idea) but IMHO it make easier to spot this is the continuation of the previous line unicode string.